corehq/apps/sms/filters.py

```diff
@@ -38,6 +38,18 @@ def options(self):
         return options_var
 
 
+class ErrorCodeFilter(BaseMultipleOptionFilter):
+    label = ugettext_noop('Error')
```

Comment:
Which error is this really?
I am confused by this when showing the status the code says that this was not an error

https://github.com/dimagi/commcare-hq/blob/6d905c1ddaca1f9077643274d57b062afe32d50f/corehq/apps/reports/standard/message_event_display.py#L28-L34

and then later again checks if the sms had an error

https://github.com/dimagi/commcare-hq/blob/6d905c1ddaca1f9077643274d57b062afe32d50f/corehq/apps/reports/standard/message_event_display.py#L80-L85

Which makes me think that even if i search for status completed, there might still be an error message in the status column.

Reply:
It'll show any error attached to either the event or the subevent. I could imagine it being possible for an event to have an error code but not be in an error status - say, if one subevent had an error but another didn't.